scaleway/table_scaleway_iam_user.go

```diff
@@ -21,18 +21,18 @@ func tableScalewayIamUser(_ context.Context) *plugin.Table {
 			Hydrate: listIamUsers,
 			KeyColumns: []*plugin.KeyColumn{
 				{
-					Name:    "id",
+					Name:    "user_id",
 					Require: plugin.Optional,
 				},
 			},
 		},
 		Get: &plugin.GetConfig{
 			Hydrate:    getIamUser,
-			KeyColumns: plugin.SingleColumn("id"),
+			KeyColumns: plugin.SingleColumn("user_id"),
 		},
 		Columns: []*plugin.Column{
 			{
-				Name:        "id",
+				Name:        "user_id",
```

Comment:
@jplanckeel Why is this being renamed? This would be a breaking change, so we usually deprecate and replace before removing to avoid breaking existing queries.

Reply:
i change this because this comment in destination PR 

https://github.com/turbot/steampipe-plugin-scaleway/pull/30#discussion_r1302905842

Reply:
We don't have BC because table is not merged in main.

Reply:
@jplanckeel Got it, thanks. I'd recommend then:
- Remove the `KeyColumns` block from the `ListConfig`, since if a user specifies `id` in a query, the plugin will use the `GetConfig` call anyway
- Keep this column name as `id`, since this is the field name we get back from the Scaleway API
- Update the `GetConfig` `KeyColumns` to use `id` again